### PR TITLE
Feature/add example for async

### PR DESCRIPTION
**Pull request by w-site**
https://github.com/w-site/react-app/issues/12

### CODE REVIEW

File: src/application/components/Home/store.js [hunk 1]
```diff
@@ -1,5 +1,28 @@
 import Store from "../../common/StoreClass";
+import { get, save } from "../../common/api";
 
-class HomeStore extends Store {}
+class HomeStore extends Store {
+  async loadHomePageClicksCount() {
```
Comment (Vasso) [inline src/application/components/Home/store.js]: store will contains actions all time or this is a temporary solution?

Comment (w-site) [reply src/application/components/Home/store.js]: why not?

Comment (w-site) [reply src/application/components/Home/store.js]: for parse data we can create some kind of model, need to think about this

**Comment by w-site** [comment]
image

**Pull request merged**